mem-01-b.xml

```diff
@@ -7629,13 +7629,15 @@ Ein besonderer Dank richtet sich an Marc Okrand ({marq 'oqranD:n:name}) für das
 
 Программирование под Android: {De'vID:n:name}
 Flutter (iOS) port: Daniel Dadap
-Клингоно-агнглийская информация: {De'vID:n:name,nolink}, with help from others
-Переводы на немецкий язык: {Quvar:n:name} (Lieven L. Litaer)
+Клингоно-агнглийская информация: {De'vID:n:name,nolink}, с помощью других
+Переводы на немецкий язык: {Quvar:n:name} (Ливен Л. Литар)
 Переводы на шведский язык: {Klingonska Akademien:url:http://klingonska.org/}
-TNG {pIqaD:n} font: Admiral {qurgh lungqIj:n:name,nolink} of the {Klingon Assault Group:url:http://www.kag.org/}
-DSC {pIqaD:n:nolink} font: {Quvar:n:name,nolink} (Lieven L. Litaer)
+Переводы на русский язык: tlhIngan (Кирилл Шилов), De'nIl (Дэниел Дэдеп), De'vID (De'vID:n:name) и Sirglin (Ольга Косарева)
```

Comment:
Did you want this line added in the other languages also?

Also, I think my name can be removed. I didn't make any contributions to the Russian data, really.

Reply:
Yes. I want this line added in other languages.